playbooks/libstatic.yml

```diff
@@ -0,0 +1,20 @@
+---
+# by default this playbook runs in the staging environment
+# to run in qa, pass '-e runtime_env=qa'
+# to run in production, pass '-e runtime_env=production'
+- name: Build libstatic infrastructure
+  hosts: libstatic_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  roles:
+    - role: libstatic
+      when: runtime_env | default('staging') == "production"
```

Comment:
Do we need this `when`?  I think this playbook should always run the `libstatic` role.

Reply:
You're right! @acozine and I added the datadog role and moved the `when` to that.